### ISSUE TITLE
Get deterministic output (same seed)

### ISSUE THREAD

**Issue by adamerose** [suno-ai/bark#540]
Is it possible to get deterministic results so the same input prompt always gives the same audio? I don't see any way to set the seed.

**Comment by JonathanFly** [comment]
Yes, but it's not that useful because if you change the input prompt even a little it can change the output a lot, so at that point, you may as well just save the audio or the generated tokens itself.
